app/src/main/java/org/oppia/app/home/WelcomeViewModel.kt

```diff
@@ -1,9 +1,9 @@
 package org.oppia.app.home
 
-import androidx.databinding.ObservableField
 import androidx.lifecycle.ViewModel
 
 /** [ViewModel] for welcome text in home screen. */
 class WelcomeViewModel : HomeItemViewModel() {
   var profileName : String = ""
+  var greeting : String = ""
```

Comment:
Prefer converting these strings  to `val` by making them observable field or pass them directly in the constructor.

Reply:
+1

Reply:
Done.